_posts/en/newsletters/2021-05-05-newsletter.md

```diff
@@ -0,0 +1,191 @@
+  other processes and communicate with them using [Cap'n Proto][capn proto].
+  These features are currently only used for testing, but the [next PR][Bitcoin
+  Core #10102] in the project will allow Bitcoin Core to be started in
+  multiprocess mode with the `bitcoin-core` process spawning separate
+  `bitcoin-wallet` and `bitcoin-gui` processes.
+
+- [Bitcoin Core #19521][] nearly completes the [coin statistics index project][bitcoin
+  core coinstats] to dramatically speed up the `gettxoutsetinfo` RPC.  Until
+  now, the RPC defaulted to scanning the full UTXO set every time it was called, making it
+  difficult for users to continually and quickly verify the coin supply or
+  compare UTXO set hashes between different nodes.
+  Users can now start their nodes with the `-coinstatsindex`
+  configuration option to begin building the coin statistics index in the background.  Once
+  it is synced, `gettxoutsetinfo` runs nearly instantly and users can specify a
+  height or block hash for the statistics.  Being able to get the
+  statistics for a particular block will be especially useful for
+  allowing community verification of [assumeUTXO][topic assumeutxo]
+  chainstate archives.
+
+- [Bitcoin Core #21009][] removes the RewindBlockIndex logic triggered when
+  updating a pre-segwit node (v0.13.0 or older) to a segwit-enforcing
+  version. Pre-segwit nodes only processed stripped blocks that lacked the (segregated) witness
+  data. The RewindBlockIndex logic discarded its copies of such blocks,
+  re-downloaded them in their complete form, and validated them using segwit's rules. As pre-segwit
+  nodes have been end-of-life since 2018, the scenario has become uncommon.
+  Future releases will instead prompt the user to reindex for an equivalent
+  outcome.
+
+- [LND #5159][] builds on [previous work][news144 lnd ampsendpayment] to add
+  support for making spontaneous Atomic Multipath Payments (AMPs) by manually
+  specifying payment parameters to the `sendpayment` RPC. Invoking `sendpayment`
+  with an AMP invoice is expected to be implemented in followup PRs.
+
+- [Rust-Lightning #893][] only allows accepting a payment if it included
+  a payment secret.  Payment secrets are created by the receiver and
+  included in invoices.  The spender includes the payment secret in
+  their payment in order to prevent attempts by third parties to reduce
+  the privacy of [multipath payments][topic multipath payments].  Alongside
+  this change are several API changes designed to reduce the chance that
+  a payment will be accepted incorrectly.
+
+- [BIPs #1104][] updates the [BIP341][] specification of [taproot][topic
+  taproot] with activation parameters based on the Speedy Trial proposal
+  (see [Newsletter #139][news139 speedy trial]).
+
+## Footnotes
+
+[^missing-peer]:
+    The data loss protection protocol, and other proposed methods such
+    as [covert requests for mutual channel closes][news128 covert] requires your channel peer to
```

Comment:
- add comma or alternatively, s/requires/require/
- consider s/peer/peers/ as they are referred to as "they" in the next sentence

```suggestion
    as [covert requests for mutual channel closes][news128 covert], requires your channel peers to
```